docker/accounts/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15
+FROM golang:1.17.1
```

Comment:
This should probably be synced with 1.16.7 right? That is the version we are using everywhere else?

Reply:
I'm not using any 1.17 features in `accounts` and the minor performance boost is irrelevant for this project, so keeping it in sync with `skyd`'s containers makes sense.